variable/rounding.cpp

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Samuel Jones
+
+#include "scipp/core/element/rounding.h"
+#include "scipp/variable/rounding.h"
+#include "scipp/variable/transform.h"
+
+Variable floor(const Variable &var) {
+  return transform(var, scipp::core::element::floor, "floor");
```

Comment:
Unless you need special flags to `transform` this does not need to be hand-written, can use `cmake` functions to generate this. Need to decide what to about variance to understand which solution you need.